earth_enterprise/src/common/performancelogger.cpp

```diff
@@ -38,22 +40,32 @@ void PerformanceLogger::log(
     const size_t size) {
 
   const timespec duration = timespecDiff(endTime, startTime);
-  const pid_t tid = syscall(SYS_gettid);
   stringstream message;
+  pthread_t tid = pthread_self();
 
   message.setf(ios_base::fixed, ios_base::floatfield);
-  message << setprecision(9)
-          << operation << " " << object << ": "
-          << "start time: " << startTime
-          << ", "
-          << "end time: " << endTime
-          << ", "
-          << "duration: " << duration
-          << ", "
-          << "thread: " << tid;
-  if (size > 0) {
-    message << ", size: " << size;
-  }
+  message << tid << ", "
+          << startTime << ", "
+          << endTime << ", "
+          << duration << ", "
+          << size << ", "
+          << operation << ", "
+          << object;
 
   notify(NFY_NOTICE, "%s\n", message.str().c_str());
```

Comment:
Need to change this to call do_notify

Reply:
This will also involve creating a file to hold the output. We want the file name to be different for every execution, which you can do by appending a timestamp or something like that.

Reply:
Hey Daniel - any updates to this comment?